coverage/lcov-report/base.css

```diff
@@ -0,0 +1,223 @@
+body, html {
+  margin:0; padding: 0;
```

Comment:
Nothing wrong here, but not sure what lcov-report/base.css is. If it belongs cool. My instinct is coverage folder shouldn't be included in your project repo